src/aws-cpp-sdk-core/source/Aws.cpp

```diff
@@ -210,14 +210,14 @@ namespace Aws
 
         Aws::Config::CleanupConfigAndCredentialsCacheManager();
 
+        Aws::Client::CoreErrorsMapper::CleanupCoreErrorsMapper();
```

Comment:
We should also be initializing the logger and installing it before we initialize the CRT, so we log the CRT startup log messages as well.  This will also prove the logger has no dependencies on the CRT, and then the mechanism(s) to cleanly shutdown with delays and quiesces around logging can all be removed.